mitm.default.ini

```diff
@@ -0,0 +1,167 @@
+# This is a sample PyRDP MITM configuration file.  The file should be
+# located in the system's user configuration directory and settings
+# will be used unless overridden at the command line.
+#
+# On most Linux distributions, this file should be in
+#
+#     $HOME/.config/pyrdp/mitm.ini
+#
+# On Windows systems, the file should be in one of:
+#
+#     %APPDATA%/<AppName>
```

Comment:
Aligning examples with the linux version which specifies both path and filename:
```suggestion
# On Windows systems, the file should be:
#
#     %APPDATA%/pyrdp/mitm.ini
```